regression/get_datasets.py

```
@@ -899,9 +916,24 @@ def download_mip_dataset(name, dir):
 datasets_path = sys.argv[1]
 dataset_type = sys.argv[2]
 
+failed = []
 if dataset_type == "lp":
     for name in LPFeasibleMittelmannSet:
-        download_lp_dataset(name, datasets_path)
+        try:
+            download_lp_dataset(name, datasets_path)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)